src/python/tritonfrontend/_api/_kservegrpc.py

```diff
@@ -81,13 +82,19 @@ class Options:
         ] = Grpc_compression_level.NONE
         infer_allocation_pool_size: int = Field(8, ge=0)
         forward_header_pattern: str = ""
-        # DLIS-7215: Add restricted protocol support
-        # restricted_protocols: str = ""
+        restricted_protocols: RestrictedFeatures = RestrictedFeatures()
```

Comment:
@GuanLuo thoughts on calling it something uniform for both GRPC and HTTP?

ex: instead of `restricted_protocols` and `restricted_apis`, how about using `restricted_apis` or `restricted_features` for both?

Reply:
Protocol / feature is a bit general but it is fine

Reply:
Modified `KServeHttp.Options` and `KServeGrpc.Options` use `Options(restricted_features=...)`